### ISSUE TITLE
Crash on ESP32 S3 + SSD1306

### ISSUE THREAD

**Issue by leelx78** [davetcc/tcMenu#411]
Hi Dave,

just realised you updated TCMenus and I gave it a go. Impressive new features!

Unfortunately I cannot run any examples on ESP32 S3 and SSD1306. Then I made a very simple new project and I get the same problem, see below.

My .ino file is

```
#include "t2_menu.h"

void setup() {
  Serial.begin(115200);
  Serial.println("init I2C");
  
  Wire.begin(5,4);

  delay(50);

  Serial.println("init");
    setupMenu();

}

void loop() {
    taskManager.runLoop();

}
```

When I compile it there are no errors but when I run it I get this:

```
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0xc (RTC_SW_CPU_RST),boot:0x8 (SPI_FAST_FLASH_BOOT)
Saved PC:0x40377b04
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3808,len:0x44c
load:0x403c9700,len:0xbd8
load:0x403cc700,len:0x2a80
entry 0x403c98d0
init I2C
init
stage1
stage1.1
stage2
stage3
E (10839) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
E (10839) task_wdt:  - IDLE (CPU 0)
E (10839) task_wdt: Tasks currently running:
E (10839) task_wdt: CPU 0: loopTask
E (10839) task_wdt: CPU 1: IDLE
E (10839) task_wdt: Aborting.

abort() was called at PC 0x420152fc on core 0


Backtrace: 0x40377ef6:0x3fc93a60 0x4037b299:0x3fc93a80 0x40380c59:0x3fc93aa0 0x420152fc:0x3fc93b20 0x40378ffd:0x3fc93b40 0x400559dd:0x3fcebe90 |<-CORRUPTED




ELF file SHA256: 8380c2ef2ac9ae4a

Rebooting...
```

  And it keeps looping that way. Note that stage x.y is some text I added to track the setup function, i.e.:
```
 void setupMenu() {
    // First we set up eeprom and authentication (if needed).
      Serial.println("stage1");

    setSizeBasedEEPROMStorageEnabled(true);
    Serial.println("stage1.1");
    // Code generated by plugins.
    gfx.begin();
    Serial.println("stage2");
    renderer.setUpdatesPerSecond(10);
    switches.init(internalDigitalIo(), SWITCHES_POLL_EVERYTHING, true);
    menuMgr.initForEncoder(&renderer, &menuNewSubMenu, 38, 39, 37);
    renderer.setTitleMode(BaseGraphicalRenderer::TITLE_FIRST_ROW);
    renderer.setUseSliderForAnalog(false);
    installMonoInverseTitleTheme(renderer, MenuFontDef(nullptr, 1), MenuFontDef(nullptr, 1), true);
    Serial.println("stage3");
}
```  

So basically from what I see the setup is done correctly but then once it switches to TaskManager it hangs. Any insight on this? I'll attach the full test project if you need it.

Thanks a lot!
[t2.zip](https://github.com/davetcc/tcMenu/files/13070937/t2.zip)

**Comment by leelx78** [comment]
Sorry my bad! Compilation errors with events on core 0 and Arduino on core 1! Putting both on Core 1 works now thanks!
